shopware/administration/6.4/bin/build-administration.sh

```diff
@@ -21,6 +36,7 @@ fi
 
 # build admin
 [[ ${SHOPWARE_SKIP_BUNDLE_DUMP-""} ]] || "${BIN_TOOL}" bundle:dump
+bin/console feature:dump || true
```

Comment:
this probably should then also be 
```suggestion
"${BIN_TOOL}" feature:dump || true
```
right? 

i guess this line is in all 4 scripts right now